fresque/lib/views.py

```diff
@@ -98,3 +111,68 @@ def user_reviews(db, username):
             Reviewer.reviewer_name == username
         ).order_by(Review.date_start).all()
     return Result({"reviews": reviews})
+
+
+def create_git_repo(db, name, desc, gitfolder, data=None):
+    ''' Create a git project based on the package information
+        and intialise a template spec file.
+    '''
+    package = db.query(Package).filter_by(name=name).all()
+
+    gitrepo = os.path.join(gitfolder, '%s.git' % name)
+    if os.path.exists(gitrepo):
+        raise 'The project repo "%s" already exists' % name
+
+    gitObj = pygit2.init_repository(gitrepo, bare=False)
+
+    # add spec template and update HEAD
+    if add_spec_template(gitObj, name, data):
+        print("spec template successfully added") # later we will use logger
+
+    return 'Project "%s" git respository created' % name
+
+
+# Create spec template on given information
+def create__spec_template(name=None, summary=None, version="0.1.0"):
+    data = {
+        'Name': name,
+        'Version': '0.0.1',
+        'Summary': summary
+    }
+    return json.dumps(data)
+
+
+def add_spec_template(gitObj, name, data):
```

Comment:
I see the idea, but I would expect the packager to have already built their package before making it available for review, so they will have a spec file already filled and therefore I do not really see why we should create dummy spec file that will never be used :)